app/assets/stylesheets/components/c-featured_section.scss

```diff
@@ -0,0 +1,37 @@
+.c-section__featured {
+  max-width: 100%;
+  background-color: #252525;
+  color: #f2f2f2;
+  clear: both;
+}
+
+.c-header__divider--featured {
```

Comment:
This should be in the header component, card below should be in the card component